prisma/schema.prisma

```diff
@@ -23,16 +23,17 @@ model Sentence {
 }
 
 model Transcription {
-    id         Int        @id @default(autoincrement())
-    sentences  Sentence[]
-    createdAt  DateTime   @default(now())
-    updatedAt  DateTime   @updatedAt
-    post       Post?
-    postId     Int        @unique
-    title      String?
-    keywords   String?
-    categories String?
-    speaker    String?
+  id               Int          @id @default(autoincrement())
+  sentences        Sentence[]
+  createdAt        DateTime     @default(now())
+  updatedAt        DateTime     @updatedAt
+  post             Post?
+  postId           Int          @unique
+  title            String?
+  keywords         String?
+  categories       String?
+  speaker          String?
+  votes            Vote[]
```

Comment:
This shouldn't be needed, as the `Vote` model contains all the information.
Gathering all the votes for a video should be done in SQL, like 
```
SELECT * FROM Vote WHERE Vote.transcription = transcription.id
```